features/registerHealthCertificate.feature

```diff
@@ -0,0 +1,68 @@
+@RegistryService
+Feature: Register Health Certificate
+
+    @RegisterHealthCertificate
+    Scenario: Submit Valid Register Health Certificate Transaction
+        Given I make a POST request to /
+            And I set json body to the file at ./features/fixtures/registerHealthCertificate/Bundle-ProvideDDCCDocument.json
+        When I receive a response
+        Then I expect response should have a status 200
+            # Check that an entry exists in response for each entry in request, in same order
+            And I expect a response entry exists for each request entry List, DocumentReference, Binary, Patient
+            And I expect response should validate against the profile http://profiles.ihe.net/ITI/MHD/StructureDefinition/IHE.MHD.ProvideDocumentBundleResponse
+            # Check each entry in response has a response location
+            And I expect response should have a json schema
```

Comment:
Good call, I agree it can probably be removed - I haven't really worked with profiles until now so I didn't realize this was already covered in the profile